session/src/test/java/org/apache/iotdb/session/SessionUT.java

```diff
@@ -244,4 +245,25 @@ public void createDeviceTemplate() throws IoTDBConnectionException, StatementExe
         "template1", schemaNames, measurementList, dataTypeList, encodingList, compressionTypes);
     session.setSchemaTemplate("template1", "root.sg.1");
   }
+
+  @Test
+  public void testBuilder() {
+    Session.Builder builder = new Session.Builder("localhost", 1234);
```

Comment:
hi
I found this implementation to be inconsistent with the use cases on jira.
We need to return a session through sessionbuild.
Session session = new SessionBuilder().build()

Reply:
Hi, thanks for the review! I think the implementation is consistent (see https://dzone.com/articles/design-patterns-the-builder-pattern). When using the pattern, however, we usually do not keep the interim builder object. I have changed the test case to make this more explicit.

Reply:
default of constructors is no parameter.
// default
Session session = new SessionBuilder().build()
SessionPool sessionpool = new SessionPoolBuilder().build()

Reply:
Correct me if I am wrong: `host` and `rpcPort` are compulsory fields for session and has no default values, so they are placed in the constructor. Same goes for sessionpool. If they are not put in constructor, users might forget to set values for them.

Reply:
ok